src/main/java/life/qbic/io/commandline/PostmanCommandLineOptions.java

```diff
@@ -103,10 +105,15 @@ void listDatasets()
   public String as_url = "https://qbis.qbic.uni-tuebingen.de/openbis/openbis";
 
   @Option(
-          names = {"-dss", "--dss_url"},
-          description = "DataStoreServer URL",
-          scope = CommandLine.ScopeType.INHERIT)
-  public String dss_url = "https://qbis.qbic.uni-tuebingen.de/datastore_server";
+      names = {"-dss", "--dss_url"},
```

Comment:
singular?

Reply:
yes, can't change without breaking the api